Containerfile

```diff
@@ -0,0 +1,26 @@
+# Start rust build process
+FROM docker.io/library/rust:latest as builder
+
+# Set working directory to /src, to keep things clean
+WORKDIR /src
+
+# Copy source files to /src
+COPY . /src
+
+# Build
+RUN cargo install --path=./ && cargo clean -v
+
+# Swap to fedora runtime for the final result
+FROM fedora:latest
```

Comment:
Better to use a stable tag to avoid unexpected changes. Also Im not familiar with Fedora so Id prefer Debian (or Ubuntu).